add example inputs to yaml and help (#431)
rapid7/insightconnect-plugins: whois/help.md

# Version History

> * 1.0.8 - Add example inputs to yaml and help.md

Reviewer: ```suggestion
* 1.0.8 - Add example inputs to plugin.spec.yaml and help.md
```

> * 1.0.8 - Add example inputs to plugin.spec.yaml and help.md

Reviewer 2: Style we should use

```suggestion
* 1.0.8 - Add example inputs
```

> * 2.0.0 - Add example inputs. Fix capitalization with `last_updated` output.

Reviewer: ```suggestion
* 2.0.0 - Add example inputs | Fix capitalization with `last_updated` output title.
```

> * 2.0.0 - Add example inputs. Fix capitalization with `last_updated` output.

Reviewer 2: We use `|` as the separator. It's something we just standardized on.

```suggestion
* 2.0.0 - Add example inputs. Fix capitalization in the title of the `last_updated` output.
```
